### PR TITLE
Deprecation documentation

### PR DESCRIPTION
**Pull request by erzetpe**

### CODE REVIEW

File: docs/home/DEPRECATION-NOTE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+The functionality that Logstash is serving in Epiphany - exporting csv can right now be achieved using Open Distro for Elasticsearch plugin.
+Logstash in this version is prone to log4j issue and this is also the reason why should be removed.
+
+Planed removal: Epiphany 2.0
+Issue: [2833](https://github.com/epiphany-platform/epiphany/issues/2833)
+
+- Hashicorp Vault
+
+The role related to Hashicorp Vault has never been developed beyond MVP/PoC stage and cannot be used for production usage and causes troubles with new Kubernetes versions.
```
Comment (romsok24) [inline docs/home/DEPRECATION-NOTE.md]: Pls reformat the sentence in order it not contain 3x 'and' word.

File: docs/home/DEPRECATION-NOTE.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Planed removal: Epiphany 2.0
+Issue: [2834](https://github.com/epiphany-platform/epiphany/issues/2834)
+
+- Istio
+
+We are considering replacement of Istio component with different solutions and Istio causes problems during upgrade of Kubernetes.
```
Comment (romsok24) [inline docs/home/DEPRECATION-NOTE.md]: _We are considering replacement of Istio component with different solution. Additionally, Istio causes problems during upgrade of Kubernetes._

File: docs/home/DEPRECATION-NOTE.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Hashicorp Vault
+
+The role related to Hashicorp Vault has never been developed beyond MVP/PoC stage and cannot be used for production usage and causes troubles with new Kubernetes versions.
+
+Planed removal: Epiphany 2.0
+Issue: [2834](https://github.com/epiphany-platform/epiphany/issues/2834)
+
+- Istio
+
+We are considering replacement of Istio component with the different solutions and Istio causes problems during upgrade of Kubernetes.
```
Comment (romsok24) [inline docs/home/DEPRECATION-NOTE.md]: _We are considering replacement of Istio component with different solution. Additionally, Istio causes problems during upgrade of Kubernetes._

**Review by romsok24 — DISMISSED** [review]

File: docs/home/DEPRECATION-NOTE.md (hunk 1 above)
Comment (erzetpe) [reply docs/home/DEPRECATION-NOTE.md]: Done.

File: docs/home/DEPRECATION-NOTE.md (hunk 2 above)
Comment (erzetpe) [reply docs/home/DEPRECATION-NOTE.md]: Done.

File: docs/home/DEPRECATION-NOTE.md (hunk 3 above)
Comment (erzetpe) [reply docs/home/DEPRECATION-NOTE.md]: Done.

**Review by romsok24 — DISMISSED** [review]

**Review by plirglo — DISMISSED** [review]

**Review by romsok24 — APPROVED** [review]

**Review by plirglo — APPROVED** [review]

**Pull request merged**